sdk/cosmos/azure-cosmos-benchmark/src/main/java/com/azure/cosmos/benchmark/linkedin/CollectionResourceManager.java

```diff
@@ -53,45 +57,52 @@ public void createResources() throws CosmosException {
         final CosmosAsyncDatabase database = _client.getDatabase(_configuration.getDatabaseId());
         final CollectionAttributes collectionAttributes = _entityConfiguration.collectionAttributes();
         try {
-            LOGGER.info("Creating container {} in the database {}", containerName, database.getId());
-            final CosmosContainerProperties containerProperties =
-                new CosmosContainerProperties(containerName, Constants.PARTITION_KEY_PATH)
-                    .setIndexingPolicy(collectionAttributes.indexingPolicy());
-            database.createContainerIfNotExists(containerProperties)
-                .block(RESOURCE_CRUD_WAIT_TIME);
+            LOGGER.info("Checking for container {} in the database {}", containerName, database.getId());
+            final CosmosAsyncContainer container = database.getContainer(containerName);
+            final Optional<CosmosContainerResponse> response = getContainerProperties(container);
+            if (response.isPresent()) {
```

Comment:
Please check for status code 200 as well

Reply:
The presence/not present is handled with getContainerProperties. Any checks will have to happen there. Resolving this convo and moving to the other open conversation.